hivemind/dht/validation.py

```diff
@@ -52,3 +53,30 @@ def strip_value(self, record: DHTRecord) -> bytes:
         """
 
         return record.value
+
+
+class CompositeValidator(RecordValidatorBase):
+    def __init__(self, validators: Dict[str, RecordValidatorBase]=None):
+        self._validators = validators if validators is not None else {}
+
+    def set_if_not_present(self, validators: Dict[str, RecordValidatorBase]) -> None:
+        for key, val in validators.items():
```

Comment:
Here and below: `val` is too ambiguous (could be expanded both to value and validator), I'd write `value` explicitly